buildbot.mariadb.org/util/nginx.conf

```diff
@@ -22,8 +22,8 @@ server {
 	ssl_certificate_key /etc/letsencrypt/live/buildbot.mariadb.org/privkey.pem;
 	# put a one day session timeout for websockets to stay longer
 	ssl_session_cache   shared:SSL:10m;
-	ssl_session_timeout 20m;
-	# force https
+	ssl_session_timeout 1d;
```

Comment:
Did you verify that specifying `1d` works? My suggestion was to use `1440m`, since I don't see anything else than minute values in nginx documentation:
http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_session_timeout

Reply:
According to `nginx` syntax [1], yes you can use it.
If couldn't, one would obtain an error for test configuration run `nginx -t`. 
[1] http://nginx.org/en/docs/syntax.html

Reply:
Cool, merged!